roles/sahara/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+sahara:
+  enabled: True
+  juno_url: http://tarballs.openstack.org/sahara/sahara-stable-juno.tar.gz
+  dashboard_url: http://tarballs.openstack.org/sahara-dashboard/sahara-dashboard-master.tar.gz
+  cafile: "{{ ssl.cafile|default('/etc/ssl/certs/ca-certificates.crt') }}"
+logs:
+    logging:
+    debug: True
+    verbose: True
```

Comment:
these should default to false as well. You can set them to true in the group_vars for the test enviornment.